File: Intermediate/4.Solve the Fibonacci series using recursion/Solution.py
def fibonacci(n):
    if n <= 1:
        return n
    return fibonacci(n - 1) + fibonacci(n - 2)

# Input from the user
n = int(input("Enter the number of terms: "))

# Output the Fibonacci series
print("Fibonacci series:")
for i in range(n):
    print(fibonacci(i), end=" ")
print()

File: Intermediate/4.Solve the Fibonacci series using recursion/Solution.java
import java.util.Scanner;

public class FibonacciRecursion {
    static int fibonacci(int n) {
        if (n <= 1) return n;
        return fibonacci(n - 1) + fibonacci(n - 2);
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter the number of terms: ");
        int n = scanner.nextInt();

        System.out.println("Fibonacci series:");
        for (int i = 0; i < n; i++) {
            System.out.print(fibonacci(i) + " ");
        }
        System.out.println();

        scanner.close();
    }
}
